src/ui/components/NetworkMonitor/Filter.tsx

```diff
@@ -0,0 +1,73 @@
+import classNames from "classnames";
+import { useState } from "react";
+
+import MaterialIcon from "../shared/MaterialIcon";
+import Icon from "../shared/Icon";
+
+import { CanonicalRequestType, RequestTypeOptions } from "./utils";
+
+export function Filter({
+  children,
+  types,
+  toggleType,
+  table,
+}: {
+  children: React.ReactNode;
+  types: Set<CanonicalRequestType>;
+  toggleType: (type: CanonicalRequestType) => void;
+  table: any;
+}) {
+  const [filterOpen, setFilterOpen] = useState(false);
+
+  return (
+    <>
+      <div className="flex items-center bg-themeTextFieldBgcolor p-1">
+        <button
+          onClick={() => setFilterOpen(!filterOpen)}
+          className={classNames("mr-1 flex items-center", {
+            "text-primaryAccent hover:text-primaryAccentHover focus:text-primaryAccentHover":
+              types.size > 0,
+          })}
+        >
+          <MaterialIcon iconSize="lg" outlined={true}>
+            filter_alt
+          </MaterialIcon>
+        </button>
+
+        <MaterialIcon iconSize="lg">search</MaterialIcon>
+
+        <input
```

Comment:
It'd be nice if this input got bumped so that it clears the expanded drawer.

Currently:
<img width="761" alt="image" src="https://user-images.githubusercontent.com/15959269/171530732-9c625b19-d0cc-4080-9f82-777cc1e443af.png">
In the console:
<img width="782" alt="image" src="https://user-images.githubusercontent.com/15959269/171530756-c57566b7-bec8-4106-9478-88339cad6ebb.png">

Reply:
Good catch! Yeah, we should push this to more consistent. I'll get this cleaned up tomorrow!